nixops/backends/libvirtd.py

```diff
@@ -222,18 +243,19 @@ def start(self):
             self.private_ipv4 = self._parse_ip()
         else:
             self.log("starting...")
-            self._logged_exec(["virsh", "-c", "qemu:///system", "start", self.vm_id])
+            self.dom.create()
```

Comment:
Oh, not "start"? Weird API.

Reply:
yes it's weird https://www.redhat.com/archives/libvir-list/2010-June/msg00140.html